scripts/inline_comments_data_collection.py

```diff
@@ -176,19 +177,24 @@ def process_comments(patch_threshold, diff_length_threshold):
                 }
                 yield data
 
+        if patch_threshold == 0:
+            continue
+
         patch_count += 1
         if patch_count >= patch_threshold:
             break
 
 
 def main():
     os.makedirs("patches", exist_ok=True)
-    os.makedirs("dataset", exist_ok=True)
-    dataset_file_path = "dataset/inline_comment_dataset2.json"
-    with open(dataset_file_path, "a") as dataset_file_handle:
-        for data in process_comments(patch_threshold=1000, diff_length_threshold=5000):
+    os.makedirs("data", exist_ok=True)
+
+    with open(phabricator.FIXED_COMMENTS_DB, "a") as dataset_file_handle:
+        for data in process_comments(patch_threshold=0, diff_length_threshold=0):
             dataset_file_handle.write(json.dumps(data) + "\n")
```

Comment:
```suggestion
            dataset_file_handle.write(orjson.dumps(elem) + b"\n")
```

Note: you need to import `orjson` :)

Reply:
Done here: 67f101e11c53ec6b8fcf75a1cd4eb7e341a8d4cf